app/components/avatar_component.rb

```diff
@@ -20,7 +24,11 @@ def image_url
   end
 
   def initials
-    "#{user.first_name[0]}#{user.last_name[0]}".upcase
+    if pet
```

Comment:
Maybe we can just use the first letter since the names are all singular (no second names)